### ISSUE TITLE
Doesn't work. Not saving MP3

### ISSUE THREAD

**Issue by chaoscreater** [rajpdus/MeetingSummarizer#1]
A couple of issues. It doesn't read the OPEN_API_KEY environmental variable. I've restarted the Python shell and it's still the same. Had to manually hardcode my API key in your script.

After that is resolved, I then tried to start recording as "test.mp3" into D:\ drive. It shows recording started and when I stopped it, nothing gets created. Tried this a few times and it's the same thing.

![image](https://github.com/rajpdus/MeetingSummarizer/assets/18227319/60cb9788-d9ea-4a13-b1ba-0e85f281efaa)

**Comment by hacksider** [comment]
Same issue here

**Comment by Kekehuaite** [comment]
@chaoscreater For the OPEN_API_KEY you have to create a ".env" file with your API Key. like : 
```.env
OPEN_API_KEY=$yourapikey
```
are you sure you have installed all the requierements ? 

```bash
foo@bar:~$ pip install -r requirements.txt
```

**Comment by chaoscreater** [comment]
> @chaoscreater For the OPEN_API_KEY you have to create a ".env" file with your API Key. like :
> 
> ```dotenv
> OPEN_API_KEY=$yourapikey
> ```
> 
> are you sure you have installed all the requierements ?
> 
> ```shell
> foo@bar:~$ pip install -r requirements.txt
> ```

Yes I installed all the requirements. It wouldn't even have been able to run if I'm missing the packages.

**Comment by drammen94** [comment]
I assume youre on Windows. You need to change the microhone input in cli.py

**Comment by chaoscreater** [comment]
> I assume youre on Windows. You need to change the microhone input in [cli.py](http://cli.py)

Yes, I'm on Windows. I've searched for "microphone" or anything related to microphone in that file and don't see anything that I could change. Could you provide the line number, or an example of what that change looks like please?

**Comment by drammen94** [comment]
in stream. Mine looks like this

# The command to record audio using FFmpeg on macOS
        stream = (
            ffmpeg
            .input("audio=Headset (Jabra Hands-Free AG Audio)", f="dshow")
            .output(output_filename, acodec="libmp3lame", format="mp3")
            .overwrite_output()

**Comment by R4ksh4** [comment]
I have the same problem on Windows11.

**Comment by Pixmotic** [comment]
Sadly got the same problem here on window 10
